### DISCUSSION TITLE
Welcome to easystats Discussions!

### DISCUSSION THREAD

**Discussion by DominiqueMakowski** [easystats/easystats#90] [General]
Let's enable this new feature and see how it coexists with the Issues

**Comment by mattansb** [comment]
How does this differ?

**Comment by DominiqueMakowski** [comment]
https://youtu.be/AiWjanAdD3s?t=53

It seems like it's to separate discussions and like questions and things like that from actual issues (i.e., bugs or in-active-developpment stuff)... but from a discussion you can open an issue

**Comment by DominiqueMakowski** [comment]
it's basically a forum, and issues are for issues... and it's true that we have a ton of discussions scattered around in issues / commits / PRs so maybe it's a good in-principle idea to centralize them... Though it could be that the reason why we are scattered is just an emerging phenomenon from our parrallel work workflow... but we'll see 😅

**Comment by strengejacke** [comment]
So please start with converting these issues:

https://github.com/easystats/bayestestR/issues?q=is%3Aissue+is%3Aopen+label%3A%22Theoretical+contemplations+%3Athinking%3A%22

into discussions. 😬

**Reply by strengejacke** [reply]
you can actually do that!

**Reply by mattansb** [reply]
How can I enable Discussions on effect size? (Why am I not admin?? Have I not proven my loyalty??)

**Reply by DominiqueMakowski** [reply]
I thought you were haha

**Reply by mattansb** [reply]
And now to delete all the projects 😈....
